src/plugins/intel_npu/src/compiler_adapter/include/izero_adapter.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ze_graph_ext.h>
+
+#include "intel_npu/common/igraph.hpp"
```

Comment:
Do we need this only for "CompiledNetwork"? If a method from this class really needs to return a "CompilerNetwork" then it should be defined here and we should remove the dependency on igraph.
A Driver/Plugin adapter would use this zero_adapter to retrieve handle/meta/binaries from the driver and construct a graph object out of them.

Reply:
removed